File: src/Node/models/apiConfigRequest.js
// apiConfigRequest.js
export class ApiConfigRequest {
  constructor({
    elementLocation,
    zoom = 0.8,
    bgImageBase64,
    bgImageBlurLevel,
    bgImageUrl,
    bgColor,
  }) {
    this.elementLocation = elementLocation; // "center", "bottomCenter", "aspectRatio"
    this.zoom = zoom;
    this.bgImageBase64 = bgImageBase64;
    this.bgImageBlurLevel = bgImageBlurLevel; // possible values 1, 2, 3
    this.bgImageUrl = bgImageUrl;
    this.bgColor = bgColor;
  }
}

File: src/CSharp/Img2Design.Api.RemoveBackground.Sample/Model/ApiConfigRequest.cs
namespace Img2Design.Api.RemoveBackground.Sample.Model
{
    public class ApiConfigRequest
    {
        public string ElementLocation { get; set; } // center, bottomCenter, aspectRatio
        public double Zoom { get; set; } = 0.8;
        public string BgImageBase64 { get; set; }
        public int BgImageBlurLevel { get; set; } // possible value 1, 2, 3
        public string BgImageUrl { get; set; }
        public string BgColor { get; set; }
    }
}
